packages/application/style/skiplink.css

```diff
@@ -6,6 +6,10 @@
 
 .jp-skiplink-wrapper {
   overflow: visible;
+
+  /* override strict containment added via Lumino PR
+   [#506](https://github.com/jupyterlab/lumino/pull/506) */
+  contain: none !important;
```

Comment:
@krassowski this CSS change needs specific scrutiny and review by you because I'm not sure if this has downstream performance implications that would undo a lot of the performance work you've done.

Reply:
Thank you @gabalafou sorry I missed it earlier. Relaxing `contain` locally for leaf-level widgets is fine, but most likely it does should still not be set to `none`. `strict` means `size layout paint style` so one should first replace it with these four and then remove one by one to see which can stay and which to remove. As there are few children in skiplink it would not matter much though.

Reply:
@t03857785 just pinging you directly to ensure that you saw the suggestion above.

Reply:
I am taking a look at the moment to determine which option(s) work as intended

Reply:
I have updated this to the most restrictive contain possible which still has the correct rendering